examples/advanced/sklearn-linear/job.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This code shows how to use NVIDIA FLARE Job Recipe to run federated learning with scikit-learn
+linear models using FedAvg algorithm.
+"""
+
+import argparse
+
+from nvflare.app_opt.sklearn.recipes.fedavg import SklearnFedAvgRecipe
```

Comment:
if we already added those imports/exports in the init file,
here we should be able to "from nvflare.app_opt.sklearn import SklearnFedAvgRecipe"